backend/routers/v1/holdings.py

```
@@ -0,0 +1,131 @@
+
+    return holding
+
+
+@router.post(
+    "/",
+    response_model=db.schema.HoldingRead,
+    status_code=status.HTTP_201_CREATED
+)
+def create_holding(
+    holding: db.schema.HoldingCreate,
+    session: Session = Depends(get_session)
```

Ruff (0.11.9)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)